demo_waterfall.py

```diff
@@ -45,70 +46,73 @@
 FREQ_INC_FINE = 0.1e6
 GAIN_INC = 5
 
-class Waterfall(object):
+
+class Waterfall:
     keyboard_buffer = []
     shift_key_down = False
-    image_buffer = -100*np.ones((NUM_BUFFERED_SWEEPS,\
-                                 NUM_SCANS_PER_SWEEP*NFFT))
+    image_buffer = -100 * np.ones((NUM_BUFFERED_SWEEPS, NUM_SCANS_PER_SWEEP * NFFT))
```

Comment:
These indentation and line-break changes however are much appreciated! (as well as the ones below)